src/pytest_cov/compat.py

```diff
@@ -3,17 +3,10 @@
 except ImportError:
     from io import StringIO
 
-import pytest
 
 StringIO  # pyflakes, this is for re-export
 
 
-if hasattr(pytest, 'hookimpl'):
-    hookwrapper = pytest.hookimpl(hookwrapper=True)
```

Comment:
`hookimpl` has been introduced in [pytest 2.8](https://docs.pytest.org/en/stable/changelog.html#id390), so this compatibility can be dropped.